src/codelabs/dart-cheatsheet.md

```diff
@@ -33,11 +33,13 @@ If the expression is an identifier, you can omit the `{}`.
 
 Here are some examples of using string interpolation:
 
+<div class="table-wrapper" markdown="1">
 | String                      | | Result |
 |-----------------------------+-+ -------|
 | `'${3 + 2}'`                | | `'5'` |
 | `'${"word".toUpperCase()}'` | | `'WORD'` |
 | `'$myObject'`               | | The value of `myObject.toString()` |
+</div>
```

Comment:
Without the extra new line in this one, it fails to render the table, because the `.table` class isn't added on the inside.

```suggestion

</div>
```